go.mod

```diff
@@ -3,19 +3,22 @@ module github.com/ibm/the-mesh-for-data
 go 1.13
 
 require (
+	cloud.google.com/go v0.66.0 // indirect
 	emperror.dev/errors v0.7.0
 	github.com/go-chi/chi v4.1.2+incompatible
 	github.com/go-chi/render v1.0.1
 	github.com/go-logr/logr v0.1.0
 	github.com/gogo/protobuf v1.3.1
-	github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf v1.4.2
```

Comment:
Incompatible with out project

Reply:
reverting ... but sooner or later it will have to be addressed as currently go.mod automatically suggests this change when building the project